homeassistant/components/moehlenhoff_alpha2/config_flow.py

```diff
@@ -0,0 +1,59 @@
+"""Alpha2 config flow."""
+import asyncio
+import logging
+
+import aiohttp
+from moehlenhoff_alpha2 import Alpha2Base
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema({"host": str})
+
+
+async def validate_input(hass: core.HomeAssistant, data):
+    """Validate the user input allows us to connect.
+
+    Data has the keys from DATA_SCHEMA with values provided by the user.
+    """
+
+    base = Alpha2Base(data["host"])
+    await base.update_data()
+
+    # Return info that you want to store in the config entry.
+    return {"title": base.name}
+
+
+class Alpha2BaseConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Möhlenhoff Alpha2 config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
```

Comment:
this has moved to manifest, remove it here.

Reply:
Removed.